src/test/java/seedu/address/storage/JsonAdaptedStudentTest.java

```diff
@@ -38,44 +39,47 @@ public void toModelType_validStudentDetails_returnsStudent() throws Exception {
     @Test
     public void toModelType_invalidName_throwsIllegalValueException() {
         JsonAdaptedStudent student =
-                new JsonAdaptedStudent(INVALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_TAGS);
+                new JsonAdaptedStudent(VALID_UUID, INVALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_TAGS);
         String expectedMessage = Name.MESSAGE_CONSTRAINTS;
         assertThrows(IllegalValueException.class, expectedMessage, student::toModelType);
     }
 
     @Test
     public void toModelType_nullName_throwsIllegalValueException() {
-        JsonAdaptedStudent student = new JsonAdaptedStudent(null, VALID_PHONE, VALID_EMAIL, VALID_TAGS);
+        JsonAdaptedStudent student =
+                new JsonAdaptedStudent(VALID_UUID, null, VALID_PHONE, VALID_EMAIL, VALID_TAGS);
         String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, Name.class.getSimpleName());
         assertThrows(IllegalValueException.class, expectedMessage, student::toModelType);
     }
 
```

Comment:
Should we include test cases for UUID? Such as `public void toModelType_nullUuid_throwsIllegalValueException()`.

Reply:
Added the following :

JsonAdaptedStudent.java
```
        if (uuid == null) {
            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, "uuid"));
        }
```

JsonAdaptedStudentTest.java
```
    @Test
    public void toModelType_invalidUuid_throwsIllegalValueException() {
        JsonAdaptedStudent student =
                new JsonAdaptedStudent(null, VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_TAGS);
        String expectedMessage = String.format(MISSING_FIELD_MESSAGE_FORMAT, "uuid");
        assertThrows(IllegalValueException.class, expectedMessage, student::toModelType);
    }
```

Since UUID is a built in java class, instead of doing something like ```throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Name.class.getSimpleName()));``` , i opted for ```throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, "uuid"));```